examples/cisco/8000e/README.md

```diff
@@ -0,0 +1,147 @@
+
+## Make sure the topology is healthy
+
+- Make sure nodes are up and running by running command `kubectl get pods -A`. The output of the command should contain namespace `cisco-ixia` and 6 nodes with status `Running`. 4 otg ports (`otg-port-*`), one otg controller (`otg-controller`), and one cisco 8000e (`8000e`) are expected to be shown if the topology is created successfully.
+  
+``` bash
+kubectl get pods  -A
+NAMESPACE            NAME                                            READY   STATUS    RESTARTS   AGE
+cisco-ixia           8000e                                           1/1     Running   0          4m28s
+cisco-ixia           otg-controller                                  2/2     Running   0          4m28s
+cisco-ixia           otg-port-eth1                                   2/2     Running   0          4m28s
+cisco-ixia           otg-port-eth2                                   2/2     Running   0          4m28s
+cisco-ixia           otg-port-eth3                                   2/2     Running   0          4m28s
+cisco-ixia           otg-port-eth4                                   2/2     Running   0          4m27s
+-- omitted -- 
+ 
+```
+
+- Make sure external ip are mapped correctly by running command `kubectl get services -n cisco-ixia`. It is expected an external ip is assigned to each of the six nodes mentioned above.  Also, the port mapping of the gnmi/gnoi/gribi/p4rt/ssh services for 8000e should match the port mapping in the topology file.  
+  
+``` bash
+kubectl get services -n cisco-ixia
+NAME                           TYPE           CLUSTER-IP      EXTERNAL-IP   PORT(S)                                                                    AGE
+service-8000e                  LoadBalancer   10.96.195.26    172.18.0.50   22:31633/TCP,9339:31543/TCP,9340:30751/TCP,9337:30331/TCP,9559:31439/TCP   7m45s
+service-gnmi-otg-controller    LoadBalancer   10.96.187.227   172.18.0.51   50051:31810/TCP                                                            7m45s
+service-grpc-otg-controller    LoadBalancer   10.96.6.73      172.18.0.52   40051:31158/TCP                                                            7m45s
+service-https-otg-controller   LoadBalancer   10.96.65.192    172.18.0.53   8443:30126/TCP                                                             7m45s
+service-otg-port-eth1          LoadBalancer   10.96.70.21     172.18.0.54   5555:30373/TCP,50071:32694/TCP                                             7m45s
+service-otg-port-eth2          LoadBalancer   10.96.166.2     172.18.0.55   5555:31205/TCP,50071:32376/TCP                                             7m45s
+service-otg-port-eth3          LoadBalancer   10.96.108.38    172.18.0.56   5555:32396/TCP,50071:30361/TCP                                             7m45s
+service-otg-port-eth4          LoadBalancer   10.96.24.228    172.18.0.57   5555:31664/TCP,50071:30416/TCP                                             7m44s
+ ```
+
```

Comment:
can you add a warning here that it can take up to X minutes to finish node boot and that the user should look into the startup.log/err if SSH and other services are still not available after the initial wait time?